src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ControlTests/Control_IsEnabled_Inheritance.xaml

```diff
@@ -0,0 +1,28 @@
+﻿<UserControl x:Class="UITests.Shared.Windows_UI_Xaml_Controls.ControlTests.Control_IsEnabled_Inheritance"
+			 xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			 xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+			 xmlns:local="using:UITests.Shared.Windows_UI_Xaml_Controls.ControlTests"
+			 xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+			 xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+			 mc:Ignorable="d"
+			 d:DesignHeight="300"
+			 d:DesignWidth="400">
+
+	<Grid>
+		<ContentControl>
+			<StackPanel>
```

Comment:
`<Grid>` and `<ContentControl>` are useless here.

Reply:
It's a test, I would assume that the `ContentControl` does have an impact for the repro ...

Reply:
Yes, the inherited `IsEnabled` from the outer `ContentControl` is part of the bug/repro.